internal/livelint/run_checks.go

```diff
@@ -5,30 +5,14 @@ import (
 	"log"
 )
 
-// Start renders the UI and listens for messages.
-func (n *Livelint) Start() {
-	err := n.ui.Start()
-	if err != nil {
-		log.Fatal(fmt.Errorf("error starting UI: %w", err))
-	}
-}
-
-// Quit quits the UI and drops back to the terminal.
-func (n *Livelint) Quit() {
-	n.ui.Quit()
-}
-
 // RunChecks checks for potential issues with a deployment.
 func (n *Livelint) RunChecks(namespace, deploymentName string, isVerbose bool) error {
-	verboseMsg := initalizeVerbose(isVerbose)
-	n.ui.Send(verboseMsg)
+	n.ui.DisplayVerboseSetting(isVerbose)
```

Comment:
What is the verbose setting actually for? As far as I could tell, all we currently do is let the user know that the verbose setting is set or not.

I would suggest making this a config on the UI implementations. We should then decide what we mean by verbose. Does it print the exact Kubernetes calls and answers? Are only certain message types written in verbose mode? Do we need some kind of message level mechanism that then is steered by the verbose flag? (probably not the last one, because then a bool wouldn't be enough)

Reply:
Yes, I agree that the verbose message in the UI is not useful for now. I removed it. Let's add it again, once the verbose mode is properly implemented.